cpp/src/ray/runtime/local_mode_ray_runtime.cc

```diff
@@ -24,9 +24,8 @@ namespace ray {
 namespace internal {
 
 LocalModeRayRuntime::LocalModeRayRuntime()
-    : worker_(ray::core::WorkerType::DRIVER,
-              ComputeDriverIdFromJob(JobID::Nil()),
-              JobID::Nil()) {
+    : job_id_(JobID::FromInt(1)),
```

Comment:
Is this job id used for anything in local mode? My understanding is that after the process dies, Ray is completely shut down. Is that right?

Reply:
Maybe add a comment /`kUnusedJobConfig`

Reply:
@wuisawesome is this code actually used?